### DISCUSSION TITLE
Alpha 188

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#7739] [Release Notes]
# Changes

- Added option to hide tabs in fullscreen mode
- Added Jazz color scheme (allan.mangan) [#7546](https://github.com/Eugeny/tabby/pull/7546)
- Disabled dynamic tab titles for SSH by default
- Mark duplicate hotkeys (Benjamin Brandmeier) [#7667](https://github.com/Eugeny/tabby/pull/7667)
- Updated bundled clink version
- Updated xterm
- Updated locales

# Fixes

- Fixed icon visibility for vertical tabs (Benjamin Brandmeier) [#7566](https://github.com/Eugeny/tabby/pull/7566)
- Deactivate Focus follows mouse as Maximize the active pane is active (Dani Nugraha) [#7591](https://github.com/Eugeny/tabby/pull/7591)
- SSH: handle Unicode characters that are chopped up between chunks
- Debounced typing in search field - #7579
- Fixed #7588 - prevent selector from opening when one is already open
- Fixed #7522, fixed #7577 - sftp: keep permissions when editing file locally

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.188'>Alpha 188</a>.</em>

**Comment by jul17** [comment]
Is tabby supported on macos m2?

**Reply by Eugeny** [reply]
Yes

**Reply by byuliia** [reply]
Which package should I use on m2?

**Reply by Eugeny** [reply]
[tabby-XXX-macos-arm64.pkg](https://github.com/Eugeny/tabby/releases/download/v1.0.188/tabby-1.0.188-macos-arm64.pkg)
